### PR TITLE
Update operations docs

### PR DESCRIPTION
**Pull request by SEJeff**
This is a set of changes I've been working on as I saw RockawayX folks ask questions when they were working on setting up a new guardian node.

### CODE REVIEW

File: docs/operations.md [hunk 1]
```diff
@@ -297,6 +379,7 @@ To run the spy against mainnet:
 
 ```bash
 docker run \
+    --pull=always \
```
Comment (barnjamin) [inline docs/operations.md:382]: Is this something that should get updated in docs site as well for starting the spy?

Comment (SEJeff) [reply docs/operations.md:382]: @barnjamin yes absolutely. When @bruce-riley merged the PR for `--env` it didn't have that and the docs as they were broke.

File: docs/operations.md [hunk 2]
```diff
@@ -193,9 +326,93 @@ alerting will be documented here.
...
+
+## Cross Chain Queries
+
+[CCQ](https://github.com/wormhole-foundation/wormhole/blob/main/whitepapers/0013_ccq.md) also known as [Wormhole Queries](https://wormhole.com/queries) is a feature to allow pulling attestations in a cross chain manner. To run ccq, a few additional flags need to be enabled on the guardian node:
+
+<!-- cspell:disable -->
+
+```shell
+--ccqEnabled=true \
+--ccqP2pPort 8996 \
```
Comment (bruce-riley) [inline docs/operations.md:407]: This can be omitted since it defaults to 8996.

**Review by bruce-riley — APPROVED** [review]

File: docs/operations.md [hunk 3]
```diff
@@ -223,6 +439,17 @@ It is safe to expose the publicWeb port on signing nodes. For better resiliency
 future guardiand releases will include listen-only mode such that multiple guardiand instances without guardian keys
 can be operated behind a load balancer.
 
+## Enabling Telemetry
+
+Optionally, the guardian can send telemetry to [grafana cloud logs](https://grafana.com/products/cloud/logs/) aka "loki".
```
Comment (evan-gray) [inline docs/operations.md:444]: "Grafana Cloud Logs"

File: docs/operations.md [hunk 4]
```diff
@@ -223,6 +439,17 @@ It is safe to expose the publicWeb port on signing nodes. For better resiliency
...
+To enable this functionality, add the following flag:
+
+```bash
+--telemetryLokiURL=$PER_GUARDIAN_LOKI_URL_WITH_TOKEN
+```
+
+New guardians should talk to the wormhole foundation to get a loki url.
```
Comment (evan-gray) [inline docs/operations.md:451]: "Wormhole Foundation", "Loki"

**Review by evan-gray — CHANGES_REQUESTED** [review]
Overall good and beneficial changes, just some grammar nits. Also there's some inconsistency in the header title casing.

File: docs/operations.md [hunk 5]
```diff
@@ -102,6 +91,143 @@ since only very few nodes support the light client protocol.
 Running a full node typically requires ~500G of SSD storage, 8G of RAM and 4-8 CPU threads (depending on clock
 frequency). Light clients have much lower hardware requirements.
 
+
+### Terra
```
[the remark this answers is not in the document]
Comment (SEJeff) [reply docs/operations.md:95]: The official name is Terra, not Terra 2, and this has a section for Terra Classic right below.

I think the correct answer to your question is no, it should not. What do you think?

File: docs/operations.md [hunk 6]
```diff
@@ -102,6 +91,143 @@ since only very few nodes support the light client protocol.
...
+
+<!-- cspell:disable -->
+
+```shell
+wormchaind tx wormhole execute-governance-vaa <guardian_set_upgrade_VAA_in_hex_format>
+```
+
+<!-- cspell:disable -->
+
+View Validator Information:
```
[the remark this answers is not in the document]
Comment (SEJeff) [reply docs/operations.md:166]: I'd consider Validator a major word and the subject, so it should be capitalized, but can lowercase information.

File: docs/operations.md [hunk 7]
```diff
@@ -102,6 +91,143 @@ since only very few nodes support the light client protocol.
...
+the transactions can be reobserved.
+
+### Cosmos / IBC Connected nodes
+
+All modern cosmos integrations happen by wormhole observing IBC transactions on gateway (wormchain). Guardian node operators do not need to run full nodes for these networks. For cosmos based chains that were added before this functionality, a full node is still necessary.
+
+The following cosmos based nodes were added prior to  gateway and guardians need to run full nodes:
+
+* Injective
+* Terra
```
[the remark this answers is not in the document]
Comment (SEJeff) [reply docs/operations.md:225]: I don't think so. I added a disclaimer to the top level Terra section to disambiguate.

File: docs/operations.md (hunk 4 above)
Comment (SEJeff) [reply docs/operations.md]: Fixed in aefd4d3432e1be115f5e92aa966517622c804a93

File: docs/operations.md (hunk 3 above)
Comment (SEJeff) [reply docs/operations.md]: Fixed in aefd4d3432e1be115f5e92aa966517622c804a93

File: docs/operations.md [hunk 8]
```diff
@@ -102,6 +91,143 @@ since only very few nodes support the light client protocol.
...
+
+Upgrade the guardian set (with a valid governance vaa):
+
+<!-- cspell:disable -->
+
+```shell
+wormchaind tx wormhole execute-governance-vaa <guardian_set_upgrade_VAA_in_hex_format>
+```
+
+<!-- cspell:disable -->
```
[the remark this answers is not in the document]
Comment (SEJeff) [reply docs/operations.md]: Nice, thanks

**Review by bruce-riley — DISMISSED** [review]

File: docs/operations.md [hunk 9]
```diff
@@ -1,38 +1,27 @@
...
-  pointing to your full node. Refer to the [Terra documentation](https://docs.terra.money/node/join-network.html)
-  on how to run a full node. From a security point of view, running only an LCD server with `--trust-node=false` pointed
-  to somebody else's full node would be sufficient, but you'd then depend on that single node for availability unless
-  you set up a load balancer pointing to a set of nodes.
-
-- **Binance Smart Chain**: Same requirements as Ethereum. Note that BSC has higher throughput than Ethereum and
-  roughly requires twice as many compute resources.
-
-**Do NOT use third-party RPC service providers** for any of the chains! You'd fully trust them, and they could lie to
+**Do NOT use third-party GRPC service providers** for any of the chains! You'd fully trust them, and they could lie to
```
Comment (evan-gray) [inline docs/operations.md:11]: why did a G get added to RPC here? not sure they're all necessarily gRPC :thinking:

Comment (SEJeff) [reply docs/operations.md]: Fixed in 0be1bf23c865e783b550bbfa6a599ed7c147d64b

**Review by bruce-riley — APPROVED** [review]

**Review by evan-gray — APPROVED** [review]

**Pull request merged**